frontend/src/slices/articles/articles.slice.ts

```diff
@@ -33,7 +35,7 @@ const { reducer, actions, name } = createSlice({
       state.dataStatus = DataStatus.FULFILLED;
     });
     builder.addCase(getArticle.fulfilled, (state, action) => {
-      state.dataStatus = DataStatus.FULFILLED;
+      state.getArticleStatus = DataStatus.FULFILLED;
```

Comment:
I guess now it should also be used for `getArticle.pending` and `getArticle.rejected`